Makefile

```diff
@@ -172,6 +181,52 @@ docker-push:
 	docker push ${DELETION_DEFENDER_IMG}
 	docker push ${UNMANAGED_DETECTOR_IMG}
 
+__tooling-image:
+	docker buildx build build/tooling \
+		--platform="$(PLATFORM)" \
+		--output=$(OUTPUT_TYPE) \
+		-t kcc-tooling
+
+__controller-gen: __tooling-image
+CONTROLLER_GEN=docker run --rm -v $(shell pwd):/wkdir kcc-tooling controller-gen
+
+.PHONY: rename-crds
+rename-crds:
+	@echo "Renaming generated CRDs..."
+	@cd $(CRD_OUTPUT_STAGING) && \
+	for file in *.yaml; do \
+		if [ "$$file" != "kustomization.yaml" ]; then \
+			base_name=$$(echo "$$file" | sed 's/apiextensions.k8s.io_v1_customresourcedefinition_//; s/.yaml$$//'); \
+			resource=$$(echo "$$base_name" | cut -d'.' -f1); \
+			domain=$$(echo "$$base_name" | cut -d'.' -f2-); \
+			new_name="apiextensions.k8s.io_v1_customresourcedefinition_$${resource}.$${domain}.yaml"; \
+			mv "$$file" "$$new_name"; \
+			echo "Renamed $$file to $$new_name"; \
+		fi \
+	done
+
+.PHONY: apis-manifests
+apis-manifests: __controller-gen
+	# Clean previous outputs
+	rm -rf $(CRD_OUTPUT_TMP)
+
+	mkdir -p $(CRD_OUTPUT_TMP)
+	$(CONTROLLER_GEN) crd:allowDangerousTypes=true paths="./apis/resources/logging/..." output:crd:artifacts:config=$(CRD_OUTPUT_TMP)
+
+	cp config/crds/kustomization_for_direct.yaml kustomization.yaml
+	kustomize edit add resource $(CRD_OUTPUT_TMP)/*.yaml
+	mkdir -p $(CRD_OUTPUT_STAGING)
+	kustomize build -o $(CRD_OUTPUT_TMP)/staging
+
+	$(MAKE) rename-crds
```

Comment:
We should see what this looks like if integrated into the existing scripts / a new script, but I don't want to block on it.

Reply:
Yes, I am parking the last couple of manual changes that need automation (via our tooling most likely) in https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/1701